src/Model/Behavior/SearchBehavior.php

```diff
@@ -42,6 +42,15 @@ class SearchBehavior extends Behavior
      */
     protected $_isSearch = false;
 
+    public function initialize(array $config)
+    {
+        parent::initialize($config);
+
+        if (isset($config['emptyValues'])) {
+            $this->_defaultConfig['emptyValues'] = $config['emptyValues'];
```

Comment:
You  need to set the key in `_config` (through `config()` method with `merge` false) not in `_defaultConfig`.